app/lib/till.rb

```diff
@@ -0,0 +1,28 @@
+require_relative "receipt"
+
+class Till
+	include Receipt
+
+	attr_reader :tax
+
+	def initialize
+		@tax = 1.0864
+	end
+
+	def total_of(order, adjustments)
+	  order_after_discount = adjustments.discount(subtotal_of(order))
+	  (order_after_discount * tax).round(2)
+	end
+
+	def subtotal_of(order)
+		order.values.collect(&:last).inject(:+)
```

Comment:
Prefer `map` over `collect`.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.